src/core/Microsoft.PowerToys.Settings.UI.Library/ViewModels/ColorPickerViewModel.cs

```diff
@@ -48,80 +63,77 @@ public ColorPickerViewModel(ISettingsUtils settingsUtils, ISettingsRepository<Ge
             SendConfigMSG = ipcMSGCallBackFunc;
         }
 
+        /// <summary>
+        /// Gets a list with all selectable <see cref="ColorRepresentationType"/>s
+        /// </summary>
+        public IReadOnlyDictionary<ColorRepresentationType, string> SelectableColorRepresentations { get; }
+
         public bool IsEnabled
         {
-            get
-            {
-                return _isEnabled;
-            }
-
+            get => _isEnabled;
             set
             {
-                if (_isEnabled != value)
+                if (_isEnabled == value)
                 {
-                    _isEnabled = value;
-                    OnPropertyChanged(nameof(IsEnabled));
+                    return;
```

Comment:
Why did you add these unnecessary returns (also into each other property?

Reply:
Based on memory, this feels like a suggested resharper tweak.

Reply:
I don't use ReSharper since version 8.0, but its the same idea.

It is to reduce the indentation of code 
and use only (when possible) flow-control on the indentation (e.g. return earlier)

```
if (_isEnabled == value){
    return;
}
```

Reply:
I understand what it does, but dont see any added value, it is just describing what will compiler do later anyway. Properties should be kept short and this just adds unnecessary lines of code

Reply:
I have remove this/change it back